Userland/Libraries/LibWeb/Crypto/Crypto.cpp

```diff
@@ -52,7 +52,7 @@ WebIDL::ExceptionOr<JS::Value> Crypto::get_random_values(JS::Value array) const
     auto& typed_array = static_cast<JS::TypedArrayBase&>(array.as_object());
 
     // 2. If the byteLength of array is greater than 65536, throw a QuotaExceededError and terminate the algorithm.
-    if (typed_array.byte_length() > 65536)
+    if (typed_array.idempotent_byte_length() > 65536)
```

Comment:
These changes seem completely unrelated to the commit message (`LibJS: Print the maxByteLength when printing a resizable ArrayBuffer`)

Reply:
Woops! Rebase shenanigans messed it up :)